### PR TITLE
edit validation in front end (signup)

### PR DESCRIPTION
**Pull request by razan99**

### CODE REVIEW

File: config.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DB_URL = postgres://eman05:12345@localhost:5432/freelance
+SECRET= eman
```
Comment (yahyaHB) [inline config.env]: this file shouldn't be here

Comment (yahyaHB) [inline config.env]: this file shouldn't be here!

File: public/js/dom.js [hunk 2]
```diff
@@ -26,39 +26,15 @@ select('#close-btn-project').addEventListener('click',()=>{
...
-  
-   const btnn= select(".buttonS");
-   btnn.addEventListener("click", function(){
-   const name = select("#name").value;
-   const password = select("#password").value;
-  
+
    fetch('/','POST' ,name, password, function(res){
      msg.textContent=res;
+      console.log("ee")
```
Comment (yahyaHB) [inline public/js/dom.js]: remove the console log from all the files

File: src/app.js [hunk 3]
```diff
@@ -1,28 +1,31 @@
-const express = require("express");
+const express = require('express');
 // const favicon = require("serve-favicon");
```
Comment (yahyaHB) [inline src/app.js]: no commented lines should be

File: src/controllers/checkUser.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+//         next();
+//       }
+//       console.log('decoded', decoded);
+//       req.userName = decoded.userName;
+//       req.userId = decoded.userId;
+//       req.loggedIn = true;
+//     });
+//   }
+//   next()
+// };
```
Comment (yahyaHB) [inline src/controllers/checkUser.js]: what this file does and why is it all commented?

**Comment by yahyaHB** [comment]
https://github.com/FACG4/Arab-per-hour/pull/32/checks?check_run_id=1585179

Have you seen the link or the results?
or at least tried to know what is the problem with travis?

File: config.env (hunk 1 above)
Comment (razan99) [reply config.env]: we did it

Comment (razan99) [reply config.env]: solveed

**Pull request merged**